src/resources/locales/fr.json

```diff
@@ -8,6 +8,11 @@
       "devToolsLabel": "Outils pour les développeurs de thèmes",
       "hideInstalledLabel": "Masquer ceux étant installés lors de la navigation",
       "colourShiftLabel": "Changer de couleure chaque minutes",
+      "albumArtBasedColors": "Changement des couleures basé sur les pochettes d'albums",
+      "albumArtBasedColorsMode": "Mode de schéma de couleure (ColorApi)",
+      "albumArtBasedColorsVibrancy": "Couleure saisie depuis les pochettes d'albums",
+      "albumArtBasedColorsVibrancyToolTip": "Désaturé: La couleur qui est la plus proéminente mais avec beaucoup moins de luminosité\nVibrations Claires: La couleur la plus vibrante, mais avec une luminosité un peu plus forte\nPrometteur: La couleur qui ressort le plus dans la pochette de l'album\nVibrations: La couleur la plus vibrante dans la pochette de l\'album",
+      "almbumArtColorsModeToolTip": "Monochrome foncé: une palette de couleurs basée directement sur la couleur principale sélectionnée, en utilisant différentes nuances de la couleur principale et en mélangeant des gris pour créer une palette de couleurs, c'est l\'inverse du monochrome clair.\nMonochrome clair: Une palette de couleurs basée directement sur la couleur principale sélectionnée, en utilisant différentes nuances de la couleur principale et en mélangeant les gris pour créer une palette de couleurs. L'arrière-plan d'un monochrome clair sera le premier plan ou la couleur du texte d'un monochrome foncé et vice versa.\nAnalogique: Schéma de couleurs basé sur la couleur principale sélectionnée, utilisant les couleurs adjacentes à la couleur principale sur le cercle chromatique.\nAnalogique complémentaire: Un schéma de couleurs basé sur la couleur principale sélectionnée, utilisant les couleurs adjacentes à la couleur principale sur le cercle chromatique et la couleur complémentaire.\nTriade: Un schéma de couleurs basé sur la couleur principale sélectionnée, utilisant les couleurs équidistantes de la couleur principale sur le cercle chromatique.\nQuad: Un schéma de couleurs basé sur la couleur principale sélectionnée, utilisant les couleurs du cercle chromatique qui sont à 90 degrés de la couleur principale.",
```

Comment:
Typo in the property name: `almbumArtColorsModeToolTip` should be `albumArtColorsModeToolTip`